hardware_layer/hardware_imu/src/dcm.cpp

```diff
@@ -18,10 +18,10 @@ void output_angles()
   }
   else if (output_format == OUTPUT__FORMAT_TEXT)
   {
-    // printf(" YPR=");
```

Comment:
this is already commented out! you should delete this, along with any useless `if else` and if other useless variables. Learn to refactor code properly.